src/main/resources/data/retail/transactions-currentaccount.json

```diff
@@ -0,0 +1,77 @@
+[
```

Comment:
These are all transfers between accounts, so counter parties cannot be merchants.

Reply:
changed it